### ISSUE TITLE
Unexpected value encountered for GetSummaryReportAsync

### ISSUE THREAD

**Issue by VincentVerweij** [Morasiu/Clockify.Net#60]
Hi there,

I did write a short program to fetch my summary report from Clockify.
As a `SummaryReportRequest` I am using this:
```csharp
SummaryReportRequest summaryReportRequest = new()
{
    DateRangeStart = new DateTime(2022, 2, 7),
    DateRangeEnd = new DateTime(2022, 2, 13),
    SummaryFilter = new SummaryFilterDto()
    {
        Groups = new List<GroupType>()
        {
            GroupType.DATE,
            GroupType.TAG
        },
        SortColumn = SortColumnType.GROUP
    },
    ExportType = ExportType.JSON
};

var reportResponse = await clockifyClient.GetSummaryReportAsync(workspace.Id, summaryReportRequest);
```

The `reportResponse` is not successful and gives ErrorMessage: Unexpected character encountered while parsing value: [. Path 'groupOne[0].children[0]._id', line 1, position 185.
The error's position and the example might slightly differ, as I did not paste the exact JSON response (truncated it a bit).

When looking at the JSON response, I have something like this:
```json
{
    "totals": [{
        "_id": "",
        "totalTime": 39784,
        "entriesCount": 45,
        "totalAmount": null
    }],
    "groupOne": [{
        "duration": 39784,
        "_id": "2022-02-07",
        "name": "2022-02-07",
        "children": [
	{
	    "duration": 24669,
	    "_id": ["61aa2fa49e0fb0338dac807b"],
	    "name": "Project A",
	    "nameLowerCase": "project a"
	}, {
	    "duration": 10940,
	    "_id": ["61aa2ff01799d3570fc4ab44"],
	    "name": "Project B",
	    "nameLowerCase": "project B"
	}, {
	    "duration": 1184,
	    "_id": ["61aa31531799d3570fc4c648"],
	    "name": "Internal meeting",
	    "nameLowerCase": "internal meeting"
	}, {
	    "duration": 2991,
	    "_id": ["61aa305d9e0fb0338dac8d05"],
	    "name": "Support tag",
	    "nameLowerCase": "support tag"
	}]
    }]
}
```

I believe it has something to do with `Clockify.Net/Models/Reports/SummaryGroup.cs`
In there the property `Id` is a string, and I believe in this case it shouldn't be.
As you can see in the JSON above, the _id fields in the groupOne array contain brackets around the IDs.
I am not sure whether it is always like that, hence logging this issue.

Apart from that, thanks for writing this nice wrapper!

**Comment by Morasiu** [comment]
Thanks I will look into that. Maybe Clockify changed their API

**Comment by VincentVerweij** [comment]
If you look into the [summary report from Clockify's API](https://clockify.me/developers-api#operation--v1-workspaces--workspaceId--reports-summary-post), we can see that its response is without the brackets, so it could be related to the fact that I use `GroupType.TAG` in the `SummaryReportRequest`. As you can select multiple tags for a time entry, it might be that they return this as an array.

Would be nice if they always returned an array for the ID and not only if you specify TAG.

**Comment by Morasiu** [comment]
Fixed in 1.15.0. Thanks for pointing that out!
